Deployment.yaml

```diff
@@ -0,0 +1,92 @@
+# deployment.yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: module-14
+spec:
+  selector:
+    matchLabels:
+      app: module-14
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: module-14
+    spec:
+      containers:
+        - name: todo-app
+          image: todo-app:prod
+          imagePullPolicy: Never
+          ports:
+          - containerPort: 5000
+          env:
+#          envFrom:
+#          - secretRef:
+#              name: app-secret5
+          - name: DB_NAME  
+            valueFrom: 
+              secretKeyRef: 
+                name: app-secret5 
+                key: DB_NAME  
+          - name: COLLECTION_NAME 
+            valueFrom: 
+              secretKeyRef: 
+                name: app-secret5 
+                key: COLLECTION_NAME 
+          - name: CLIENTID 
+            valueFrom: 
+              secretKeyRef: 
+                name: app-secret5 
+                key: CLIENTID 
+          - name: CLIENTSECRET 
+            valueFrom: 
+              secretKeyRef: 
+                name: app-secret5 
+                key: CLIENTSECRET 
+          - name: LOGGLY_TOKEN 
+            valueFrom: 
+              secretKeyRef: 
+                name: app-secret5 
+                key: LOGGLY_TOKEN
+          - name:  SECRET_KEY 
+            valueFrom: 
+              secretKeyRef: 
+                name: app-secret5 
+                key:  SECRET_KEY       
+          - name:  FLASK_APP 
+            valueFrom: 
+              secretKeyRef: 
+                name: app-secret5 
+                key:  FLASK_APP
+          - name:  FLASK_ENV 
+            valueFrom: 
+              secretKeyRef: 
+                name: app-secret5 
+                key:  FLASK_ENV
+          - name:  LOG_LEVEL
+            valueFrom: 
+              secretKeyRef: 
+                name: app-secret5 
+                key:  LOG_LEVEL
+          - name:  URL 
+            valueFrom: 
+              secretKeyRef: 
+                name: app-secret5 
+                key:  URL
+          - name:  DOCKER_REGISTRY_SERVER_USERNAME 
+            valueFrom: 
+              secretKeyRef: 
+                name: app-secret5 
+                key:  DOCKER_REGISTRY_SERVER_USERNAME
+          - name:  DOCKER_REGISTRY_SERVER_PASSWORD 
+            valueFrom: 
+              secretKeyRef: 
+                name: app-secret5 
+                key:  DOCKER_REGISTRY_SERVER_PASSWORD
```

Comment:
Does your app actually need your DockerHub username/password? I removed these and the app still works very happily!

Reply:
This was when trying to work out why it was not working so i will remove